unit-tests/src/test/java/org/eclipse/collections/impl/block/factory/ComparatorsTest.java

```diff
@@ -201,11 +201,11 @@ public void fromFunctionsWithTwoArgs()
     {
         Person raab = new Person("Don", "Raab", 0);
         Person white = new Person("Barry", "White", 0);
-        Person manilow = new Person("Barry", "Manilow", 0);
```

Comment:
Most of the "narrow scope" fixes are great. But this is one where I felt it was more readable before than after, and I was reluctant to make this inspection a breaking one.

Reply:
I can suppress it.